### PR TITLE
Mastery Exercise submission 

### PR DESCRIPTION
**Pull request by mmeadow3**

### CODE REVIEW

**Comment by mmeadow3** [comment]
@morecallan
DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE! DONE!

File: word-search.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const limit = require("./limit-ten")
+const transformStream = Transform()
+
+
+
+let file = fs.createReadStream('/usr/share/dict/words')
+  .pipe(es.split())
+  .pipe(es.map(transformStream._transform = (buffer, callback) => {
+	 if (buffer.toString().startsWith((process.argv[2].toString()).toLowerCase())) {
+	 	let rightWord = (buffer.toString())
```
Comment (morecallan) [inline word-search.js]: You aren't using this rightWord var anywhere, are you?

**Comment by morecallan** [comment]
Looking good, @mmeadow3!
-Nice commit history: broken into chunks similar to requirement. +1 coolPoints.
-Some nice es6'in in there with the destructuring of the stream object and the .startsWith. +1 coolPoints

Before you merge:
-Note that the map IS a transform stream and does not need to be passed a transform stream. Take a look at the docs on it. Your functionality will stay the same inside of it, you just don't need to wrap it in a transform stream. 
-You aren't QUITE handling all the case insensitivity. Both the word you input AND the word in the list need to be case insensitive, eg: search: turd - get Turd or turd. 
-Comments!

**Comment by mmeadow3** [comment]
@morecallan 
Fixed!

**Comment by morecallan** [comment]
@mmeadow3 

Oops, now it's entirely case-sensitive! Make it so it's 100% case insensitive. Sorry if my comment on it was misleading. You were 50% of the way there before.

**Comment by morecallan** [comment]
Once you fix that, you're ready to merge!

**Pull request merged**